demo.js

```diff
@@ -1,12 +1,61 @@
 import DanceParty from './src/p5.dance';
 import jazzy_beats from './metadata/jazzy_beats';
+const DanceAPI = require('./src/api');
+const interpreted = require('raw-loader!./src/p5.dance.interpreted.js');
+
+/**
+ * Inject p5.dance.interpreted code directly (this is only acceptable because we
+ * don't have untrusted user input in our dev env). Also attach all functionNames
+ * to the global namespace so that they can be called from the console, making
+ * for easier testing
+ */
+function injectInterpreted(api) {
+  // This list is not exhaustive, and more can/should be added as needed
+  const functionNames = [
+    'atTimestamp',
+    'everySeconds',
+    'everySecondsRange',
+    'everyVerseChorus',
+    'runUserSetup',
+    'runUserEvents',
+    'getCueList',
+  ];
+
+  const globals = new DanceAPI(api);
+  const code = interpreted + `return {${functionNames.join(',')}};`;
+
+  const params = [];
+  const args = [];
+  for (let k of Object.keys(globals)) {
+    params.push(k);
+    args.push(globals[k]);
+  }
+  params.push(code);
+  const ctor = function () {
+    return Function.apply(this, params);
+  };
+  ctor.prototype = Function.prototype;
+  const result = new ctor().apply(null, args);
+
+  // inject onto window
+  functionNames.forEach(name => window[name] = result[name]);
```

Comment:
potentially contentious?

could also stick it onto something like `window.interpreted.atTimestamp`

Reply:
Yeah, I think I like the parallel between `nativeAPI.` and `interpretedAPI.`

Reply:
Looking at this further, I think it might not be worthwhile to expose these functions anywhere. It looks like if I add something like `atTimestamp` I still need to call `getCueList` afterwards to get things working properly.

It might make more sense to just say that if you want to step through custom code, you stick it in the `// Sample user code:` section of demo.js rather than trying to do things from the chroms console.